src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,158 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * The Ui object deals with interactions with the user, including on screen messages and taking in user input.
+ */
+public class Ui {
+    private Scanner scanner;
+
+    /**
+     * Initializes the Ui object with a new Scanner object.
+     */
+    public Ui() {
+        scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Gets the user input.
+     *
+     * @return User input.
+     */
+    public String getUserInput() {
+        return scanner.nextLine();
+    }
+
+    /**
+     * Shows the greeting message.
+     */
+    public String showGreeting() {
+        String message = "Eh what's up\n"
+                + "What do you want?";
+        return wrapMessage(message);
+    }
+
+    /**
+     * Shows the goodbye message.
+     */
+    public String showGoodbye() {
+        String message = "Alright I'll see you around!\n";
+        return wrapMessage(message);
+    }
+
+    /**
+     * Shows the loading file error.
+     */
+    public String showLoadingError() {
+        return (wrapMessage("Error loading file!"));
+    }
+
+    /**
+     * Shows the list of tasks in the list.
+     *
+     * @param tasks Tasks currently in the list.
+     */
+    public String showList(TaskList tasks) {
+        String message = "Here are the tasks in your list:\n";
+        for (int i = 0; i < tasks.getSize(); i++) {
+            Task task = tasks.getTask(i);
+            message += (i + 1)
+                    + ". "
+                    + task
+                    + "\n";
+        }
+        return wrapMessage(message);
+    }
+
+    /**
+     * Shows the add message when the user adds a task.
+     *
+     * @param task The added task.
+     * @param tasks The current list of tasks.
+     */
+    public String showAdd(Task task, TaskList tasks) {
```

Comment:
name should accurately reflect the method as per coding standard. Do you think it is better to rephrase to "showAddMessage"?